### ISSUE TITLE
Radiotap: UnsupportedVersion

### ISSUE THREAD

**Issue by bogu88** [Ragnt/AngryOxide#38]
Trying to run it on a Pi 4 gives this error: Error   | Couldn't read packet data with Radiotap: UnsupportedVersion. I built from source using the steps listed.

**Expected behavior**
Works normally running on a laptop on Kali, but installing on rpi gives the error Radiotap: UnsupportedVersion. As well as 0 ERs.

** Hardware (please complete the following information):**
 - Device: Raspberry Pi 4
 - OS: DietPi
 - Interface: Alfa AWUS036NHA

Thanks

**Comment by Ragnt** [comment]
Likely that network manager is controlling the interface still. Ensure it is not running, or anything else that may try and control the interface (such as dhcpcd).

**Comment by bogu88** [comment]
So stopping dhcpd did solve the error, but still getting bad performance out of the pi with 0 ERs, so I'm assuming its just a problem with cpu speed or sd card performance is too slow.

**Comment by Ragnt** [comment]
Yeah, CPU speed is likely the culprit, optimization is an issue on this version when it comes to embedded systems. Headless mode helps there. I’m building the next version from the ground with embedded optimization in mind.

**Comment by olivervbk** [comment]
I have the same issue:
`Couldn't read packet data with Radiotap: UnsupportedVersion`

Device: Raspberry Pi 4
OS: ArchlinuxARM (6.9.8-2-aarch64-ARCH)
wifi: AWUS036ACM (wlan1)

I have `iwd` running, but it shouldn't be managing the wlan1 interface...

The interesting this is that I was testing this using `tshark`, and looking at the pcapng I see the radiotap data included:
`Radiotap Header v0, Length 30`

`tshark` seems to create a monitor interface `mon0` and uses that.
At one point `iwconfig wlan1 mode monitor` also seems to have created `mon0` but then it stopped creating it, idk.
Tried with `airmon-ng start wlan1`, which seems to rename the interface to `wlan1mon`, and tried to use that for angryoxide, but then things seg faulted and borked the driver/network stack.

**Comment by Ragnt** [comment]
That should be classified as a warning instead of as an Error, probably.

This usually happens (I think) because the driver is sending frames without radiotap headers to the buffer, but usually only the first few are effected. After that the rest come through fine.

If this is happening during the middle of running, then it almost certainly is iwd placing the device back into station mode.

TLDR; this is safe to ignore if you get it at the beginning of the runtime.

You don’t need to worry about the creation of a VIF for monitor mode. Tshark and Airmon do that, but we don’t. We set the interface into monitor mode directly. AO won’t start unless the driver is reporting that it went into monitor mode.
